src/olympia/scanners/tests/test_views.py

```diff
@@ -73,67 +74,33 @@ def test_get(self):
         good_result_1 = ScannerResult.objects.create(
             scanner=WAT, version=good_version_1
         )
-        VersionLog.objects.create(
-            activity_log=ActivityLog.create(
-                action=amo.LOG.APPROVE_VERSION,
-                version=good_version_1,
-                user=self.user,
-            ),
-            version=good_version_1,
+        ActivityLog.create(
+            amo.LOG.APPROVE_VERSION, good_version_1, user=self.user
```

Comment:
are all these changes also connected to the issue?

Reply:
no, that's a bit of refactoring because `ActivityLog.create()` already deals with version log.